.github/workflows/linux.yml

```diff
@@ -331,3 +325,17 @@ jobs:
 
       - name: Show ccache stats
         run: ccache --show-stats
+
+  Overall_Status:
```

Comment:
It is not required as of now (as I've removed the conditional start of this workflow) yet it could be beneficial in the future, for example for Smart CI enabling.